operator/src/main/java/oracle/kubernetes/operator/rest/model/AdmissionReview.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.rest.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import oracle.kubernetes.json.Description;
+
+/**
+ * AdmissionReview represents a request or a response object for an admission webhook call.
+ *
+ * <p></p>When a user performs an operation to a Kubernetes resource, Kubernetes ApiServer invokes all admission
+ * webhooks that have registered their interests to the resource. Upon invocation, the Kubernetes ApiServer passes an
+ * AdmissionReview object to the webhook, and expects the webhook to return an AdmissionReview as the response. An
+ * admission webhook uses the information in the AdmissionReview request to perform the necessary operation
+ * the resource, and sends back an AdmissionReview object to the ApiServer about the result of the operation.
+ * </p>
```

Comment:
OK, I think I am getting a better sense of what is going on here. If I understand you correctly, and AdmissionReview is a Kubernetes-created wrapper of either an AdminRequest, or a webhook-created wrapper of an AdmisisonResponse. Is that correct? If so, it would probably be better to say it that way rather than that K8S passes a review object _as a_ request and expects back a review object _as a_ response.